src/basic_recipes/title.jl

```diff
@@ -0,0 +1,30 @@
+@recipe(Title, titletext, plot) do scene
+    Theme(
+        textsize=40,
+        raw=true,
+        font=theme(scene, :font),
+        align = (:center, :bottom), 
+    )
+end
+
+function convert_arguments(::Type{<:Title}, titletext::AbstractString, plot::Scene)
+    @info "convert_argument called"
+    (titletext, plot)
+end
+function AbstractPlotting.plot!(t::Title)
+    @extract t (titletext, plot)
+    
+    plot = to_value(plot)
+    pos = lift(pixelarea(plot)) do area
```

Comment:
Just to confirm - is the current approach to `hbox` the plot and the title, or use `text!`?

Reply:
My plan is to stick with `hbox` until it works. More as an exercise. If it works, we can talk about changing the approach :smile: